src/main/java/io/cdap/plugin/kudu/KuduSink.java

```diff
@@ -94,18 +93,13 @@ public KuduSink(KuduSinkConfig kuduSinkConfig) {
   @Override
   public void configurePipeline(PipelineConfigurer configurer) {
     super.configurePipeline(configurer);
-    Preconditions.checkArgument(!Strings.isNullOrEmpty(kuduSinkConfig.optSchema), "Write schema is not specified. Please add" +
-      "the write schema.");
+    FailureCollector failureCollector = configurer.getStageConfigurer().getFailureCollector();
+    kuduSinkConfig.validate(failureCollector);
```

Comment:
do we need `failureCollector.getOrThrowException()` here?

Reply:
add failureCollector.getOrThrowException()